indico/web/client/js/react/components/ReviewRating.jsx

```diff
@@ -0,0 +1,52 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Rating} from 'semantic-ui-react';
+
+import './ReviewRating.module.scss';
+
+export default function ReviewRating({min, max, value, disabled, onChange}) {
+  const ratingRange = _.range(min, max + 1);
+  const suiRatingRange = _.range(1, ratingRange.length + 1);
+  const maxRating = suiRatingRange[suiRatingRange.length - 1];
```

Comment:
Nope, it is not. `maxRating` is the prop passed to  SUIR `Rating` field which operates on values between 0 and `max`. However, users can declare the rating range to be less than 0 so in case of a negative `min`, `max + 1` won't work correctly here.